### ISSUE TITLE
support chinese traditional card data

### ISSUE THREAD

**Issue by bluelovers** [mtgatool/mtgatool-desktop#9]
support chinese traditional card data

**Comment by Manuel-777** [comment]
Hi @bluelovers ! sorry for the late reply here.

I just came across the Loc files in the latest update and WIzards aparently removed Chinese from there.

I am sure these are available for Chinese players, but it seems they use a completely different build of the game?

**Comment by bluelovers** [comment]
https://gatherer.wizards.com/Pages/Card/Details.aspx?multiverseid=511970

https://scryfall.com/card/tsr/4/zhs/%E5%A4%A9%E4%BD%BF%E6%81%A9%E5%85%B8?utm_source=discord

**Comment by Manuel-777** [comment]
Yes, I know Gatherer/Scryfall both have the data available, but we need the source from MTG Arena itself, that is what we use for all other languages.

That is all managed here;
https://github.com/mtgatool/mtgatool-metadata

**Comment by bluelovers** [comment]
mtga didn't have Chinese version, but i hope can see Traditional Chinese card by English name

**Comment by Manwe-777** [comment]
As of today Chinese cards data is still not available on MTGA Loc sources; ill keep this open in case it gets added back; Just FYI, using other sources is not an option I want to consider, since I want to keep the metadata sources at a minimum, if possible, only Scryfall and MTG Arena data itself.
